README.md

```diff
@@ -1,5 +1,10 @@
-# Antenna
+# SW360 Antenna 
 
+[![Eclipse Public License 2.0](https://img.shields.io/badge/license-EPL--2.0-green.svg "Eclipse Public License 2.0")](LICENSE)
+[![Build Status](https://travis-ci.com/eclipse/antenna.svg?branch=master)](https://travis-ci.com/eclipse/antenna)
+
+![Eclipse SW360 Antenna](antenna-documentation/src/site/resources/images/logo.png)
```

Comment:
What do you think about adding the logo in the same line as the heading in the first line?

Reply:
I had that in the beginning, since I thought the same, then changed it, since it looks a bit weird.

Reply:
I tried it again by simply deleting the text in the header, since the logo does exactly the same. @bs-ondem do you think it is better now, or still to be improved?

Reply:
Imo it is better now with only the logo. :)

Reply:
@neubs-bsi The new logo seems to be to big. You can modify the width/height with the inline HTML tags.